android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -329,6 +329,65 @@ public String getCarrierSync() {
   @ReactMethod
   public void getCarrier(Promise p) { p.resolve(getCarrierSync()); }
 
+  @ReactMethod(isBlockingSynchronousMethod = true)
+  public double getTotalDiskCapacityNewSync() {
```

Comment:
Apparently there's a new API for API >= 30 (android 11) [getStorageDirectory](https://developer.android.com/reference/android/os/Environment#getStorageDirectory())

Reply:
:laughing:  never a dull moment as the APIs march along. Only 14 API advances from now we'll be able to remove that version code check ;-)